docker/bin/prep_demo.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -ex
+
+DEIS_BIN="${DEIS_BIN:-deis}"
+
+echo "Creating the demo app $DEIS_APPLICATION"
+$DEIS_BIN apps:create "$DEIS_APPLICATION" --no-remote || true
+
+echo "Configuring the new demo app"
+
+ENV_FILES=(
+  "jenkins/branches/demo/default.env"
+  "jenkins/regions/${DEIS_PROFILE}.env"
+  "jenkins/branches/${BRANCH_NAME}.env"
+)
+
+# reads which ever of the above files exist in order and combines values
+# pre-installed in jenkins
+ENV_VALUES=( $(envcat "${ENV_FILES[@]}") )
```

Comment:
@jgmize @glogiotatidis I did go ahead and publish that env file combining utility as [envcat](https://github.com/pmac/envcat). I couldn't figure out how to install it in a nice way during this script run, so I've installed it globally onto the jenkins box. Please correct me if this isn't the right way to go here.